modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -0,0 +1,136 @@
+
++
+. Create the object definition file you saved in the previous step.
++
+----
+$ oc create -f volumesnapshotclass.yaml
+----
+
+. Create a VolumeSnapshot object:
+
++
+.volumesnapshot-dynamic.yaml
+[source,yaml]
+----
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshot
+metadata:
+  name: mysnap
+spec:
+  volumeSnapshotClassName: csi-hostpath-snap <1>
+  source:
+    persistentVolumeClaimName: myclaim <2>
+----
+
+<1> The request for a particular class by the volume snapshot. If nothing is set, the default class is used if available.
++
+<2> The name of the PersistentVolumeClaim bound to a PersistentVolume. This defines what you want to create a snapshot of. Required for dynamically provisioning a snapshot.
+
++
+. Create the object definition file you saved in the previous step:
++
+----
+$ oc create -f volumesnapshot<provisioning-method>.yaml
+----
+
+
+To manually provision a snapshot:
+
+. Specify a volumeSnapshotContentName as the source for the snapshot, in addition to defining volume snapshot class as shown above.
++
+.volumesnapshot-manual.yaml
+[source,yaml]
+----
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshot
+metadata:
+  name: snapshot-demo
+spec:
+  source:
+    volumeSnapshotContentName: mycontent <1>
+----
+<1> The volumeSnapshotContentName source field is required for pre-provisioned snapshots.
+
+. Create the object definition file you saved in the previous step:
++
+----
+$ oc create -f volumesnapshot<provisioning-method>.yaml
+----
+
+Once the snapshot has been created in the cluster, additional details about the snapshot are available.
+
+. Display details about the volume snapshot that was created.
++
+----
+$ oc describe volumesnapshot<provisioning-method> mysnap
+----
+
+The following example displays additional details about the volume snapshot you created.
+
+.volumesnapshot.yaml
+[source,yaml]
+----
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshot
+metadata:
+  name: mysnap
+spec:
+  source:
+    persistentVolumeClaimName: myclaim
+  volumeSnapshotClassName: csi-hostpath-snap
+status:
+  boundVolumeSnapshotContentName: snapcontent-1af4989e-a365-4286-96f8-d5dcd65d78d6 <1>
+  creationTime: "2020-01-29T12:24:30Z" <2>
+  readyToUse: true <3>
+  restoreSize: 500Mi
+----
+<1> The volume snapshot content is the pointer to the storage itself.
+<2> The time when the snapshot was cut. The snapshot contains the volume content as available at this indicated time.
+<3> If value = `true`, the snapshot can be used to restore as a new PVC.
+  +
+If value = `false`, the snapshot has been cut. However, the storage backend needs to perform additional tasks to make the snapshot usable so that it can be restored as a new volume. For example, AWS EBS data might be moved to a different, less expensive location, which can take several minutes.
```

Comment:
I'd define AWS, and probably EBS. I've never used AWS, so I'm only vaguely familiar with EBS. Enterprise block storage? But that's a guess.